lib/src/modules/apm.dart

```diff
@@ -54,6 +61,13 @@ class APM {
   /// [String] id of the trace.
   /// [String] key of attribute.
   /// [String] value of attribute.
+  ///
+  /// Deprecated: from version v12.7.2.
+  ///
+  /// Please migrate to the App Flows APIs: [startFlow], [setFlowAttribute], and [endFlow].
+  @Deprecated(
+    'Please migrate to the App Flows APIs: APM.startAppFlow, APM.endFlow, and APM.setFlowAttribute. This feature was deprecated after <next_release_deprecation_version>',
```

Comment:
I think we need to put the actual version here and in other places where we use it.
(Same for other instances of this).

```suggestion
    'Please migrate to the App Flows APIs: APM.startAppFlow, APM.endFlow, and APM.setFlowAttribute. This feature was deprecated in v13.0.0',
```